Sqooba/cookiecutter-pypackage
# [naming] prefix packages with sqooba

To test:

```
cookiecutter https://github.com/Sqooba/cookiecutter-pypackage --checkout feature/prefix-packages-with-sqooba
```

Pay attention to the project name used, say `foo`.

To verify (in the root directory of the project):

```
pip install -r requirements_dev.txt (this will fixed in a diffent PR)
make test # should work
```

```
sqooba-foo
```
should output

```
Replace this message by putting your code into sqooba.foo.cli.main
See click documentation at http://click.pocoo.org/
```
